src/cli/cli_udp.cpp

```diff
@@ -50,6 +50,35 @@ UdpExample::UdpExample(otInstance *aInstance, OutputImplementer &aOutputImplemen
     memset(&mSocket, 0, sizeof(mSocket));
 }
 
+/**
+ * @cli udp bind
+ * @code
+ * udp bind :: 1234
+ * Done
+ * @endcode
+ * @code
+ * udp bind -u :: 1234
+ * Done
+ * @endcode
+ * @code
+ * udp bind -b :: 1234
+ * Done
+ * @endcode
+ * @cparam udp bind [@ca{netif}] @ca{ip} @ca{port}
+ * - netif: The binding network interface, which is determined as follows:
```

Comment:
Can you make each parameter on each bullet code font please

- `netif`:
- `ip`:
- `port`: